linera-core/src/unit_tests/wasm_worker_tests.rs

```diff
@@ -357,10 +360,11 @@ where
                 height: broadcast_block_height,
                 index: 0,
                 authenticated_signer: None,
-                is_skippable: false,
+                attributes: MessageAttributes::skippable(),
```

Comment:
Here, too. Or did you mean to change the test scenario?

Reply:
Nope, this is just me working in a plane :)

Reply:
same